app/views/claims/_school_search.html.erb

```diff
@@ -31,18 +31,14 @@
           <strong>No results match that search term. Try again.</strong>
         </p>
       <% else %>
-        <p class="govuk-body">
-          <span class="govuk-hint">
-            Enter the school name. Use at least four characters.
-          </span>
-        </p>
+        <span class="govuk-hint">
```

Comment:
Do these still show on seperate lines? Or are they munged together?

Reply:
Yeah we double checked and it appears exactly the same as two paragraph tags (with a gap between) 👍

Reply:
I thought I'd get the definative answer on this, set to `display: block` here:

https://github.com/alphagov/govuk-frontend/blob/2f29eadef4850627c0b76cb57fb05cc51be2fce3/src/govuk/components/hint/_hint.scss#L9

Reply:
Cool 👍